### PR TITLE
Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by rodzadra**

### CODE REVIEW

**Comment by nadar** [comment]
Thank you! Looks great! In order to fully integrate this language to LUYA there only a few files left.

https://github.com/luyadev/luya/issues/1159 <= There you can see all the files which requires to be translated, and maybe subscribe yourself to this issue so we can notify your for further translations.

Thanks for helping us making LUYA better!

File: modules/cms/src/frontend/messages/pt/cms.php [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    'tb_seo_link' => 'Link da URL',
+    'tb_seo_keywords' => 'Palavras-chaves',
+    'tb_seo_keywords_notfound' => 'Nenhuma palavra-chave encontrada! Você deve adicionar palavras-chaves para analisar seu conteúdo.',
+    'tb_seo_warning' => 'Algumas de suas palavras-chaves não foram encontradas em seu conteúdo, Para corrigir isto, você deve alterar suas palavras-chaves ou adicionando conteúdo para as palavras-chaves definidas',
+    'tb_edit_alt' => 'Edite esta tabela na interface do CMS',
+    'tb_visible_not_alt' => 'Esta página NÃO está visível para o visitante',
+    'tb_visible_alt' => 'Esta página é visível para o seu visitante',
+    'block_audio_name' => 'Áudio',
+    'block_audio_help_soundurl' => 'Coloque o código embed do soundcloud.com aqui.',
+    'block_dev_name' => 'Design-Tester',
```
Comment (nadar) [inline modules/cms/src/frontend/messages/pt/cms.php]: Below here, the translation seems to be english.

Comment (rodzadra) [inline modules/cms/src/frontend/messages/pt/cms.php]: OOps!
Done: [#1236](https://github.com/luyadev/luya/pull/1236)

**Comment by nadar** [comment]
@rodzadra So i am going to close this PR and you will commit further changes into #1236 ?

**Pull request closed without merging**